recipes/icu/all/conanfile.py

```diff
@@ -1,19 +1,24 @@
 from conan import ConanFile
 from conan.tools.apple import is_apple_os
-from conan.tools.build import cross_building
+from conan.tools.build import cross_building, stdcpp_library
 from conan.tools.env import Environment, VirtualBuildEnv
 from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, mkdir, rename, replace_in_file, rm, rmdir, save
 from conan.tools.gnu import Autotools, AutotoolsToolchain
 from conan.tools.layout import basic_layout
 from conan.tools.microsoft import is_msvc, unix_path
-from conan.tools.scm import Version
-from conans.tools import get_gnu_triplet, sha256sum, stdcpp_library
 import glob
 import os
 import shutil
+import hashlib
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=1.57.0"
```

Comment:
```suggestion
required_conan_version = ">=1.54.0"
```
DESTDIR has been fixed on 1.54, don't need to enforce to the latest version available. https://docs.conan.io/en/latest/changelog.html#id51